docs/ub_theme/README.md

```diff
@@ -0,0 +1,106 @@
+# useblocks-theme
+This repository contains the theme for all Useblock documentation sites.
+
+> **NOTE:** 
+> - Useblocks uses the Sphinx-Immaterial theme so ensure that the theme is already installed and applied.
+> - Install `sphinx_immaterial` by running this command: `pip install sphinx-immaterial`.
+> - Then activate the theme by adding it to the `extension` variable under **conf.py**: `extensions = ["sphinx_immaterial"]`
+> - And change the `html_theme` variable to `html_theme = "sphinx_immaterial"`
+
+## Installation
+To install the files from this repository, you must have [Git](https://git-scm.com) installed.
+
+* Change directory to your root directory if you are in a subdirectory.
+* Add the files from the repository as a dependency to your docs project:
+  ```bash
+  git subtree add --prefix docs/ub_theme/ https://github.com/useblocks/ub_theme.git main --squash
+  ```
+
+> **NOTE:** 
+> 
+> You can update the _ub_theme_ repository by using:
+> ```bash
+> git subtree pull --prefix docs/ub_theme/ https://github.com/useblocks/ub_theme.git main --squash
+> ```
+
+## Configuration
```

Comment:
Maybe a small chapter "update" would be also good, which explains the command to get the latest changes from ubtheme repo